.pre-commit-config.yaml

```diff
@@ -29,7 +29,6 @@ repos:
   - id: end-of-file-fixer
     exclude: >-
       ^docs/[^/]*\.svg$
-  - id: requirements-txt-fixer
```

Comment:
Instead of removing the hook, use include/exclude to point it to `.in` files only

Reply:
Done